README.md

```diff
@@ -231,9 +231,10 @@ Input can be provided in one of two formats.
 File in JSONL format, i.e., a JSON object per line with structure as defined by the `TheoryAssertionRepresentationWithLabel` class:
 ```
 {
-  "theory_statements": [<list of theory statement logical forms in prefix notation>],
-  "assertion_statement": <string containing the assertion in prefix notation>,
-  "label": <optional boolean label>
+  "theory_statements": List of theory statement logical forms in prefix notation.
+  "assertion_statement": String containing the assertion in prefix notation.
+  "label": Optional Boolean label field. input object may or may not have a label (gold label) specified, to compare accuracy against. Output object will contain a boolean value as returned from the theorem prover or None if the theorem prover threw an exception.
```

Comment:
```suggestion
  "label": Optional Boolean label field. Input object may or may not have a label (gold label) specified, to compare accuracy against. Output object will contain a boolean value as returned from the theorem prover or None if the theorem prover threw an exception.
```

This could be clearer. I was expecting something like this:

> Optional boolean field. If set with value true, then XXX.